tests/spinup_tests.rs

```diff
@@ -93,6 +93,11 @@ mod spinup_tests {
         testcases::simple_spin_rust_works(CONTROLLER).await
     }
 
+    #[tokio::test]
+    async fn simple_redis_rust_works() {
```

Comment:
Do we want one of these for HTTP too?  It's less likely to go unnoticed, but in terms of root cause it could just as easily have been the HTTP SDK that went awry.

Reply:
We already have one for HTTP: `simple_spin_rust_test`. I'll change the name so it's more clear.